addons/l10n_ma/models/res_company.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from markupsafe import Markup
+
+from odoo import api, fields, models
+
+from odoo.addons.base.models.ir_qweb_fields import nl2br
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    ice = fields.Char("ICE", help='Common Company Identifier', default='')
```

Comment:
It's a good practice to prefix fields added for a given localization with the localization module, so that when we look at the DB schemes or whatever we always easily know where that comes from.

default value for char field is already  an empty string, isn't it? :)

```suggestion
    l10n_ma_ice = fields.Char("ICE", help='Common Company Identifier')
```

Reply:
Yes, I added the default just for the sense of making it visible in the code level.
but I'll do that since this is our convention. :100:

Reply:
Why do you want empty string instead of False though?
Better to have NULL in db when possible :wink: